dali/ft/fttransform.cpp

```diff
@@ -868,7 +871,8 @@ bool TransferServer::pull()
                     LOG(MCdebugProgress, unknownJob, "Extend length of target file to %" I64F "d", lastOffset);
                 }
             }
-
+            if (out)
+                curProgress.numWrites += out->getStatistic(StNumDiskWrites);
```

Comment:
I think this should be
prevWrites = out->getStatistic(StNumDiskWrites);
 before the call to out_>write, and then
 curProgress.numWrites = out->getStatistic(StNumDiskWrites) - prevWrites
here.  I would expect it to be 0 a lot of the time because the write will be buffered but not written

Reply:
However, looking at the code I'm not sure if the output is buffered or not.  I am slightly concerned that some of the options may have almost no buffering.  e.g., if utf8 translation is going on, or other things.
There is also a call to out->flush() which might cause potentially efficiency issues.